src/lib/spec/library_spec.moon

```diff
@@ -0,0 +1,22 @@
+library = require 'library'  -- from redx
```

Comment:
This file is in the wrong directory, should be in `/spec` with all the other tests